src/errors/runtime/templates.js

```diff
@@ -123,6 +123,6 @@ export default {
     [RUNTIME_ERRORS.browserConnectionError]:               '{originErrorMessage}\n{numOfNotOpenedConnection} of {numOfAllConnections} browser connections have not been established:\n{listOfNotOpenedConnections}\n\nHints:\n{listOfHints}',
     [BrowserConnectionErrorHint.TooHighConcurrencyFactor]: 'The error can be due to a concurrency factor that is too high for the host machine’s performance (the factor value {concurrencyFactor} was specified). ' +
                                                            'Try to decrease the concurrency factor or ensure more system resources are available on the host machine.',
-    [BrowserConnectionErrorHint.UseBrowserInitOption]: 'Use the \'--browser-init-timeout\' option to allow more time for the browser to start. The timeout is set to {browserInitTimeoutMsg}.',
+    [BrowserConnectionErrorHint.UseBrowserInitOption]: 'Use the \'browserInitTimeout\' option to allow more time for the browser to start. The timeout is set to {browserInitTimeoutMsg}.',
```

Comment:
Use double quotes here:
`'Use the "browserInitTimeout"...`

Reply:
```suggestion
    [BrowserConnectionErrorHint.UseBrowserInitOption]: 'Use the \"browserInitTimeout\" option to allow more time for the browser to start. The timeout is set to {browserInitTimeoutMsg}.',
```